doc/classes/Particles2D.xml

```diff
@@ -16,12 +16,14 @@
 			<return type="Rect2">
 			</return>
 			<description>
+				Returns a [code]Rect2[/code] which contains the location and size of the rectangle where the particles are currently present.
 			</description>
 		</method>
 		<method name="restart">
 			<return type="void">
 			</return>
 			<description>
+				Restarts the particles 
```

Comment:
Missing period. Also confusing - does this restart all the existing particles, or does it restart the emitter?